CHANGELOG.md

```diff
@@ -202,6 +207,12 @@
 * Change fingerprint generation in `CodeClimateReporter.swift` to use
   the relative file path to better support CI/CD on multiple machines.  
   [HA Pors](https://github.com/hpors)
+  
+* Add `accessibility_label_for_image` rule to warn if a SwiftUI
+  Image does not have an accessibility label and is not hidden from
+  accessibility. When this is the case, the image's accessibility
+  label defaults to the name of the image file.  
+  [Ryan Cole](https://github.com/rcole34)
```

Comment:
Now, there are two entries in the CHANGELOG. You probably want to remove this one. 😉

Reply:
Ah shoot my bad, I could've sworn I cleaned up this one when rebasing but guess I missed it 😅